### ISSUE TITLE
Run multiple versions of wetty on same server

### ISSUE THREAD

**Issue by llarose40** [butlerx/wetty#160]
We are trying to set up a second version of wetty to point to a second host.  Nginx correctly directs the request to our /wetty2/public/wetty2/index.html file and uses port 3002 that is entered in the command parameters.  But the connection seems to always connect to host 1 instead of the new host 2.

**Comment by butlerx** [comment]
how are you configuring the second instance of wetty
Could you also paste got nginx config if possible

**Comment by llarose40** [comment]
Hello.  Thank you for the response.  After further review to the version that is in the repository now, we found that we are using an old version of wetty.  I believe we downloaded the version that we're using in June of 2017 and it uses hterm, not xterm.  After several internal meetings, we have decided to update the version we're using to the newest version of wetty, which I believe will have quite a few differences.  That being said, if you still want to discuss the old hterm version and where we might be going wrong, I can still provide more information.  But, I think we are going to pursue the most current version and I already think we're seeing better results.
